jupyter_rsession_proxy/__init__.py

```diff
@@ -73,9 +72,16 @@ def db_config(db_dir):
         f.close()
         return db_config_name
 
-    def _support_arg(arg):
+    def _support_args(args):
         ret = subprocess.check_output([get_rstudio_executable('rserver'), '--help'])
-        return ret.decode().find(arg) != -1
+        help_output = ret.decode()
+        return {arg: (help_output.find(arg) != -1) for arg in args}
+
+    def _get_www_frame_origin(default="same"):
+        try:
+            return os.getenv('JUPYTER_RSESSION_PROXY_WWW_FRAME_ORIGIN', default)
+        except Exception:
+            return default
```

Comment:
I don't think a try/except block is relevant here, I don't know of an error to be raised that we want that can be raised in this situation using os.getenv.

```suggestion
        return os.getenv('JUPYTER_RSESSION_PROXY_WWW_FRAME_ORIGIN', default)
```

Could you update the PR title or description to try capture this is now configurable via an environment variable? Whatever is put in the title will be visible via the changelog btw

Reply:
Ah, I've seen https://github.com/jupyterhub/jupyter-rsession-proxy/pull/148 now.

So this PR isn't really introducing this after all, its already there?